CHANGELOG.md

```diff
@@ -10,8 +10,10 @@
 - Rename `marshall` to `marshal` in all the occurrences (#2977)
 - Remove `componenterror.ErrAlreadyStarted` and `componenterror.ErrAlreadyStopped`, components should not protect against this, Service will start/stop once.
 - Rename `ApplicationStartInfo` to `BuildInfo`
+- Rename `ApplicationStartInfo.ExeName` to `BuildInfo.Command`
 
 ## 💡 Enhancements 💡
+ - Add `Description` to `BuildInfo`
```

Comment:
This is actually a rename of the `ApplicationStartInfo.LongName`

Reply:
I have updated this